packages/SwingSet/tools/run-utils.js

```diff
@@ -103,11 +105,29 @@ export const makeRunUtils = (controller, harness) => {
   // promise that can remain pending indefinitely, possibly to be settled by a
   // future message delivery.
 
+  /**
+   * `freeze` but not `harden` the proxy target so it remains trapping.
+   * Although a frozen-only object will not be defensive, since it could still
+   * be made non-trapping, these are encapsulated only within proxies that
+   * will refuse to be made non-trapping, and so can safely be shared.
+   * @see https://github.com/endojs/endo/blob/master/packages/ses/docs/preparing-for-stabilize.md
+   */
+  const objTarget = freeze({});
+
+  /**
+   * `freeze` but not `harden` the proxy target so it remains trapping.
+   * Although a frozen-only object will not be defensive, since it could still
+   * be made non-trapping, these are encapsulated only within proxies that
+   * will refuse to be made non-trapping, and so can safely be shared.
+   * @see https://github.com/endojs/endo/blob/master/packages/ses/docs/preparing-for-stabilize.md
+   */
+  const funcTarget = freeze(() => {});
+
```

Comment:
We don't need a `funcTarget` in this file, as per https://github.com/Agoric/agoric-sdk/pull/10795/files#r1907647464

```suggestion

```

Reply:
Done